### DISCUSSION TITLE
Fast Dialog Utility during Navigation and Drive

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#290] [Q&A] [answered]
I come from Kurviger #2 (2022), calculate still with Kurviger #3, but I navigate with Cruiser 2023 👍
The season is almost over and after more of 3000 km also the experiences with both tools.
I would like suggest this Dialog for Navigation during Drive with Cruiser. . .
a) that you can switch on/off the recalculation in the front end while driving
b) check of the voice announcement. . . sometimes I have the impression that the voice announcement is gone . . .
![Cruiser Dialog #3](https://github.com/devemux86/cruiser/assets/132825580/7cd20528-72c0-45e7-9e1a-4d23c2fd9388)

**Comment by WolfgangZ42** [comment, marked as answer]
Both of your suggestions are already there:
Long tap on left high navigation window:

<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/d2fa99a9-c2b8-4f8b-acbc-16c5abe6af2e"/>#

results in:

<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/41c37869-ed92-4a9b-8e89-ae28be9cb6db"/>

and you can switch on/off recalculation and get a voice test.

**Reply by SF4User** [reply]
👍👍👍 aaaahhh! Perfekt!

**Comment by devemux86** [comment]
@SF4User thanks for using Cruiser!

As @WolfgangZ42 explained, there are 2 screens that appear when we **tap** or **long press** on the top left navigation panel:
- **Tap** for the navigation **actions**:
  - Skip next waypoint
  - Skip multiple waypoints
  - Avoid roadblock
  - Route overview

- **Long press** for the navigation **options**:
  - Rerouting on / off
  - Auto zoom on / off
  - Voice guidance on / off
  - Voice test

**Reply by Cer-o** [reply]
No sabía de estas opciones y me alegra haberlas conocido. Muchas gracias.

**Reply by SF4User** [reply]
Vielen Dank Devemux. Das ist perfekt und für das Handling auf dem Motorrad sehr gut gelungen. Ich habe vor Jahren unseren Motorrad Club auf Calimoto eingestellt und versuche derzeit alle zu Cruiser herüber zu ziehen, denn Cruiser ist wirklich das perfekte Tool.

**Reply by devemux86** [reply]
@SF4User Thank you!
If you or your friends have ideas for improvements or new features, please write them so we can discuss them.

@SF4User Vielen Dank!
Wenn Sie oder Ihre Freunde Ideen für Verbesserungen oder neue Funktionen haben, schreiben Sie sie bitte auf, damit wir sie diskutieren können.

**Comment by SF4User** [comment]
Yes! I already have still a suggestion. . . or even two or three.
First: It is very difficult for a beginner to find after Install of the App the right configuration . . . and all essential Options.
You can (have to) make many, many adjustments until you have a tablet that is suitable for navigation.
Many User have not the force, to do it. Here e.g. is my setting:
![Screenshot_20230820_142726_Cruiser](https://github.com/devemux86/cruiser/assets/132825580/9536a536-2a32-4c2b-818a-e07a379067aa)

I already had the problem with Kurviger#2 App, that I had to open the app 3x . . . 4x installed and deleted - until I had the energy to get used to it. Then it was rewarded.
My suggestion: a "Default" Button in the settings. . . for the Navigation "Virgins". . . After installation, e.g. a General Setting a working configuration, automatic downloaded some local maps and POI Maps (e.g. Mapsforge?) and off we go!
A lot of my friends use the Calimoto app because you can't (or don't have to) set anything and it starts right away - and then they bother furthermore with this stuff.

I still have my problems now. . . . Mapsforge, OpenAndroMaps, rendering, rendering engine, route options GraphHopper, MapQuest, OpenRouteService, OSRAM, Valhalla, BRouter, etc. etc. etc. I have no idea what to do with it and why . . . this is all too difficult to understand.
I have an individual attitude . . . an it works perfekt (for me) and if I could save this as my “Owner Default” setting – that would be a 2nd suggestion too.
Because I won't change anything anymore - the app/navigation is working! My prescription: 
Iron Rules: Don't touch a collapsing Window/XP 😝😂

Now another revolutionary suggestion:
Would be even better to maybe completely split the Cruiser App . . . two Versions.
a) the "Standard Cruiser" with the typical functions and settings for the road (and all simplifications ... only one map type or max. two; 1x online, 1x offline) . . . so you can capture many Calimoto User.
b) a "Special Off Road Cruiser" for the off-road people (with all the BRouter stuff etc.) . . . The off-road drivers and road drivers are very specialized different (time a go, I drived self all off-road routes in the Alps - with a paper map)
Two different apps would be a great simplification, for any application - fot both User types.
I think: If you need both, you will order both . . .  😉😉

**Comment by 0709wiwiwi** [comment]
@SF4User 
![image](https://github.com/devemux86/cruiser/assets/54167201/a4d76d0d-ef9b-49c3-a9a8-ba97f65fb327)
Once [profiles ](https://github.com/devemux86/cruiser/discussions/165#discussioncomment-6761216)  are realised probably will be usefull.  All SF4user team members so should than drive with the very same similar Cruiser settings.  "Fingers crossed"

**Reply by SF4User** [reply]
THAT would be a dream! if you could transfer the toures and the settings via a QR code.
We drive in small groups, or in the large group - only 2-3 people with Navigation in operation. We all have 6.5" . . . 6.8" Samsung mobile phones - one setting would always fit.
Saving a default setting with QR code . . . that would be very, very nice.

**Reply by 0709wiwiwi** [reply]
Such a settings transfer by QR code as you do suggest is even new ;-). Track transfer designs by QR code was only mentioned once by using gpz files.  As gpz files can contain images it could be made possible to add a QR code, but this QR item has not even been discussed at all yet whether this could be usefully used or not.

**Reply by devemux86** [reply]
Using QR code to share routes, favorites, settings, etc. is an interesting idea,
as it could be done easily by users without the difficulty of transferring files.

Let's discuss this in:
- https://github.com/devemux86/cruiser/discussions/291

**Comment by devemux86** [comment]
You can all look forward to 3-4 new major features in the near future that are meant to help you make better use of Cruiser. :slightly_smiling_face:
